### PR TITLE
fix(helm): fix helm lint

### PR DESCRIPTION
**Pull request by sergioalmendrosdiaz**

### CODE REVIEW

**Comment by dmitry-midokura** [comment]
It is relatively easy to test these changes:

point your evp-chart to your local thingsboard chart and then run:

```
helm template release_name -n namespace evp-chart/
```

Does it make sense?

**Review by dmitry-midokura — DISMISSED** [review]
Please explain how did you validate this change.

File: helm/thingsboard/templates/node-db-configmap.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ data:
   SQL_ATTRIBUTES_BATCH_THREADS: {{ .Values.sql.attributes.batch.threads | quote }}
   SQL_EVENTS_BATCH_THREADS: {{ .Values.sql.events.batch.threads | quote }}
   SQL_TS_BATCH_THREADS: {{ .Values.sql.ts.batch.threads | quote }}
-  SQL_TS_LATEST_BATCH_THREADS: {{ .Values.sql.ts-latest.batch.threads | quote }}
-  QUEUE_RULEENGINE_PROMETHEUSSTATS_ENABLED: {{ .Values.rule-engine.prometheus-stats.enabled | quote }}
-  TB_QUEUE_RE_MAIN_PROCESSING_STRATEGY_TYPE: {{ .Values.rule-engine.queues.main.processing-strategy-type | quote }}
+  SQL_TS_LATEST_BATCH_THREADS: {{ .Values.sql "ts-latest" "batch" "threads" | quote }}
```
Comment (ignacio-casso) [inline helm/thingsboard/templates/node-db-configmap.yaml:54]: Don't we need the "index" keyword, like [here](https://github.com/midokura/evp-chart/blob/767c558/charts/evp/templates/properties-configmap.yaml#L12)?

**Review by ignacio-casso — CHANGES_REQUESTED** [review]

File: helm/thingsboard/templates/node-db-configmap.yaml (hunk 1 above)
Comment (sergioalmendrosdiaz) [reply helm/thingsboard/templates/node-db-configmap.yaml:54]: good catch :muscle:

**Comment by sergioalmendrosdiaz** [comment]
Done, there was another failure after the one I have fixed:

```
$ salmendros@salmendros [09:46:52] [~/midokura/dev/evp2/thingsboard-ce-k8s/helm/thingsboard] [fix_helm_lint *]
-> % helm lint .
==> Linting .

1 chart(s) linted, 0 chart(s) failed

```

**Review by ignacio-casso — APPROVED** [review]

**Pull request merged**